[FIX] Reenvio de NF-e

fabioluna opened the pull request:
Gera novo xml com a data de envio atualizada, quando é realizado
reenvio por erro da NF-e através da tela de documentos eletrônicos.

danimaribeiro commented:
Eu acho melhor colocar aqui a geração do xml novamente.
Ao clicar para voltar a provisório.

https://github.com/Trust-Code/odoo-brasil/blob/12.0/br_account_einvoice/models/invoice_eletronic.py#L457

fabioluna committed:
[FIX] Reenvio de NF-e

Gera novo xml com a data de envio atualizada, quando é realizado
reenvio por erro da NF-e através da tela de documentos eletrônicos.

danimaribeiro committed:
Merge branch '11.0' into 11.0_fix_action_send_eletronic_invoice
